microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/EnableJwtSubsystemSetupTask.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.eap.qe.microprofile.jwt;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.MicroProfileServerSetupTask;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+/**
+ * A way enable/disable MP-JWT subsystem when not enabled
+ */
+public class EnableJwtSubsystemSetupTask implements MicroProfileServerSetupTask {
+
+    private static final Address MP_JWT_EXTENSION_ADDRESS = Address
+            .extension("org.wildfly.extension.microprofile.jwt-smallrye");
+    private static final Address MP_JWT_ADDRESS = Address.subsystem("microprofile-jwt-smallrye");
+
+    private boolean wasExtensionAdded;
+    private boolean wasSubsystemAdded;
```

Comment:
Optional: For non-java native programmers might be difficult to know what is default value for `boolean` thus I prefer to initialize it explicitly to `false`.

Reply:
Thanks, will update now.